### ISSUE TITLE
FTB App not showing instances, problems with logging in, etc, reopening of a past issue that got closed

### ISSUE THREAD

**Issue by GalacticTurtle01** [FTBTeam/FTB-App#933]
### What Operating System

Windows 11

### Debug Code

FTB-DBGIXEJAQOTAB

### Describe the bug

Crashes, missing instances, login issues, visual bugs, failed installations.

### Steps to reproduce

1. Open FTB App
2a. Download a modpack (sometimes it downloads but after restart of the app it disappeared but 80% it doesn't download)
3. View the instances in the app (shows the screen that says that no instances exist
4. Backup the .ftb folder and reinstall
5. Copy and paste the instances from the backup self made .ftb folder
6. Issues still stay even after deleting the actual .ftb folder itself

### Expected behaviour

Able to view instances, download instances, no crashes

### Screenshots

![image](https://github.com/FTBTeam/FTB-App/assets/72573516/1fd51dcb-ff0f-422e-a9f7-8cc03cd530b9)

### Additional information

On the new login screen as seen in the picture, it stays stuck on the waiting for step. One time it did log me in but then the app crashed and when I reopened it I was logged out again. The same thing also happened with my instances, out of the 8 instances, I was able to see 3 of them reappear before an app crash. I was also able to install Stoneblock 3 and FTB Academy but after a crash it disappeared again so I deleted those 2 instances from my folder. Occasionally I get the error code for logging in that says invalid information 0000001 (I forgot how many 0's were there but I know it was basically just 1.

Basically every issue from my old post is still happening, more detail down below:

https://github.com/FTBTeam/FTB-App/issues/822

Thank you and have a good day
Aaron I

**Comment by MichaelHillcox** [comment]
Hi, looks like your app is a bit out of date but we've seen this kinda issue before. Can you update your app and confirm if this is still an issue?

**Comment by blockbreaker1361** [comment]
My app updated yesterday to 1.25.4, which is the latest version. When I logged in this morning all of my modpacks are gone. All that progress I made for nothing. Fix this app. This isn't the first time this has happened on you app.

**Comment by MichaelHillcox** [comment]
> My app updated yesterday to 1.25.4, which is the latest version. When I logged in this morning all of my modpacks are gone. All that progress I made for nothing. Fix this app. This isn't the first time this has happened on your app.

It's possible that is due to the settings migration system. Would you be able to share your settings.json file?
